tests/org.pitest.pitclipse.ui.tests/src/org/pitest/pitclipse/ui/behaviours/pageobjects/RefactorMenu.java

```diff
@@ -39,7 +39,7 @@ public void renameClass(String newClassName) {
 
     private void clickRename() {
         SWTBotMenuHelper menuHelper = new SWTBotMenuHelper();
-        menuHelper.findMenu(bot.menu(REFACTOR), RENAME).click();
+        menuHelper.findMenu(menuHelper.findWorkbenchMenu(bot, REFACTOR), RENAME).click();
```

Comment:
I need to understand: why is that required? The situation where this is required looks like when there's another dialog open? Doesn't this represent a bug in the test?

Reply:
This is technically not required in this particular case, but this implementation doesn't rely on the focus, which in some cases depends heavily on the window manager of the OS.

With this implementation, we just make sure to always use the correct shell to open the menu.

Reply:
Could you please describe the scenario where this is required? I seem to recall that you propose this PR because otherwise, a few tests in another PR were failing? Or did I not recall correctly?